packages/ckeditor5-editor-inline/src/inlineeditoruiview.js

```diff
@@ -217,14 +215,20 @@ export default class InlineEditorUIView extends EditorUIView {
 				return {
 					top: this._getPanelPositionTop( editableRect, panelRect ),
 					left: editableRect.left,
-					name: 'toolbar_west'
+					name: 'toolbar_west',
+					config: {
+						withArrow: false
+					}
 				};
 			},
 			( editableRect, panelRect ) => {
 				return {
 					top: this._getPanelPositionTop( editableRect, panelRect ),
 					left: editableRect.left + editableRect.width - panelRect.width,
-					name: 'toolbar_east'
+					name: 'toolbar_east',
+					config: {
+						withArrow: false
+					}
```

Comment:
Missing tests for these changes. They should land in this section https://github.com/ckeditor/ckeditor5/blob/master/packages/ckeditor5-editor-inline/tests/inlineeditoruiview.js#L205.